gui/bower.json

```diff
@@ -1,41 +0,0 @@
-{
-  "name": "angular-ndslabs",
-  "description": "AngularJS Client GUI for the NDS Labs API",
-  "version": "1.1.0",
-  "homepage": "https://github.com/nds-org/ndslabs",
-  "license": "MIT",
-  "private": true,
-  "dependencies": {
-    "angular": "~1.5.0",
-    "angular-alert": "https://github.com/bodom0015/angular-alert.git",
-    "angular-bootstrap": "~1.2.4",
-    "angular-bootstrap-pwgen": "~0.1.1",
-    "angular-busy": "~4.1.3",
-    "angular-clipboard": "~1.4.2",
-    "angular-cookies": "~1.5.0",
-    "angular-google-analytics": "~1.1.7",
-    "angular-gravatar": "~0.4.2",
-    "angular-grid": "https://github.com/bodom0015/angular-grid.git",
-    "angular-loader": "~1.5.0",
-    "angular-messages": "~1.5.0",
-    "angular-mocks": "~1.5.0",
-    "angular-resource": "~1.5.0",
-    "angular-route": "~1.5.0",
-    "angular-swagger-ui": "0.3.2",
-    "angular-wizard": "https://github.com/bodom0015/angular-wizard.git",
-    "bootstrap": "^3.3.6",
-    "font-awesome": "^4.7.0",
-    "font-awesome-animation": "~0.0.8",
-    "html5-boilerplate": "~5.2.0",
-    "lodash": "~4.5.1",
-    "ng-tags-input": "3.1.1",
-    "ng-password-strength": "https://github.com/subarroca/ng-password-strength.git",
-    "term.js": "https://github.com/chjj/term.js.git#^0.0.7",
-    "js-yaml": "^3.10.0"
-  },
-  "resolutions": {
-    "angular": "~1.5.0",
-    "angular-animate": "~1.5.0",
-    "lodash": "4.5.1"
-  }
-}
```

Comment:
`bower.json` goes away and all of these dependencies move to `package.json` instead